packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -29,16 +29,12 @@ service TemporaryConnectionRpc {
 }
 
 message MessageRef {
-  int64 timestamp = 1;
-  int32 sequenceNumber = 2;
-  string messageChainId = 3;
-  string streamId = 4;
-  int32 streamPartition = 5;
-  string publisherId = 6;
-}
-
-message ContentMessage {
-  string body = 1;
+  string streamId = 1;
+  int32 streamPartition = 2;
+  int64 timestamp = 3;
+  int32 sequenceNumber = 4;
+  bytes publisherId = 5;
+  string messageChainId = 6;
 }
 
 enum StreamMessageType {
```

Comment:
At some point we could have e.g. `ProtocolMessage` main type, which is either `ContentMessage`, or `GroupKeyRequest`, or `GroupKeyResponse`? But not part of this PR, maybe.